### PR TITLE
[bitnami/contour] fix: automatic update of ingress.status by contour

### PR DESCRIPTION
**Pull request by tricky42**
**Description of the change**
As the helm chart uses contour-envoy as the name for the envoy service (which differs from the default envoy service name 'envoy' which contour expects as default), this change adjusts the default configuration to ensure automatic ingress.status updates work out of the box.

**Benefits**
The automatic updating of the Ingress status will work with the default configuration of the Helm-Chart.

**Possible drawbacks**
none

**Checklist** 
- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [x] Title of the PR starts with chart name (e.g. `[bitnami/chart]`)
- [x] Applied changes to values to both value yaml files

### CODE REVIEW

File: bitnami/contour/values-production.yaml [hunk 1]
```diff
@@ -59,6 +59,10 @@ configInline:
   leaderelection:
     # configmap-name: leader-elect
     configmap-namespace: '{{ .Release.Namespace }}'
+  ## This sets the service name that will be inspected for address details to be applied to Ingress objects.
+  # As by default the envoy services name deployed by this helm chart is 'contour-envoy', this configuration is needed
+  # in order to ensure, that the ingress status is automatically updated by contour.
+  envoy-service-name: contour-envoy
```
Comment (joancafom) [inline bitnami/contour/values-production.yaml:65]: This is nice, but I think that the name of the Service can actually differ if, for instance, the property `.Values.fullnameOverride` is provided.

Instead, I think I would be great to use the exact same expression used to generate the name:
https://github.com/bitnami/charts/blob/6bf9dfd96083702704461d268895a1e6416fdde3/bitnami/contour/templates/envoy/service.yaml#L4-L6

```suggestion
  envoy-service-name: '{{ include "common.names.fullname" . }}-envoy'

File: bitnami/contour/values-production.yaml [hunk 2]
```diff
@@ -59,6 +59,10 @@ configInline:
   leaderelection:
     # configmap-name: leader-elect
     configmap-namespace: '{{ .Release.Namespace }}'
+  ## This sets the service name that will be inspected for address details to be applied to Ingress objects.
+  # As by default the envoy services name deployed by this helm chart is 'contour-envoy', this configuration is needed
```
Comment (joancafom) [inline bitnami/contour/values-production.yaml:63]: I would change this here to remove any mention to 'contour-envoy' and optionally say that the expression used is the same one to generate the name in the Envoy-Service

https://github.com/bitnami/charts/blob/6bf9dfd96083702704461d268895a1e6416fdde3/bitnami/contour/templates/envoy/service.yaml#L4-L6

**Review by joancafom — CHANGES_REQUESTED** [review]
Thank you so much for your PR! Could you please take at my comments?

File: bitnami/contour/values-production.yaml (hunk 2 above)
Comment (tricky42) [reply bitnami/contour/values-production.yaml]: Yes, done so.

**Comment by tricky42** [comment]
Implemented the requested changes. I think after fighting a bit to find why Contour is not updating the ingress.status although it was fixed in an earlier version of Contour, I couldn't see this better solution here ;)

**Review by tricky42 — COMMENTED** [review]
Implemented all requested changes.

**Review by tricky42 — COMMENTED** [review]
Implemented all requested changes.

**Review by joancafom — APPROVED** [review]
LGTM! Thanks for your PR 😁

**Pull request merged**